### DISCUSSION TITLE
Can't reach nginx ui on port 9000

### DISCUSSION THREAD

**Discussion by luckyluggi** [0xJacky/nginx-ui#150] [Q&A] [answered]
I installed ti on an ubuntu 22.04 VM using the install script:
`bash <(curl -L -s https://raw.githubusercontent.com/0xJacky/nginx-ui/master/install.sh) install`

When i run `service --status-all | grep ng` i only see the nginx service but not the nginx-ui service
But when i run `systemctl status nginx-ui` i see that the service is running, so i guess thats ok.

The Problem now is that i cannot open https://myip:9000 in the browser.
I opened the port in the VMs Firewall. 

When i run `netstat -tulpen` i see the service is running on port 9000:
`tcp6       0      0 :::9000                 :::*                    LISTEN      0          23011      1960/nginx-ui`
But it says tcp6. could that be the problem?

**Comment by 0xJacky** [comment, marked as answer]
Please visit http://ip:9000 not https://ip:9000

**Reply by luckyluggi** [reply]
thanks
